### ISSUE TITLE
can't instantiate an inner class

### ISSUE THREAD

**Issue by ittayd** [lihaoyi/Ammonite#520]
```
@ import $ivy.`org.apache.avro:avro:1.8.1`                           
import $ivy.$                                                        
@ new org.apache.avro.Schema.Parser                                  
Class file for org.apache.avro.Schema$Parser not found               
Compilation Failed                                                   
@ new org.apache.avro.Schema.Parser                                  
cmd1.sc:1: erroneous or inaccessible type                            
val res1 = new org.apache.avro.Schema.Parser                         
    ^                                                                
Compilation Failed                                                   
@ classOf[org.apache.avro.Schema].getClasses                         
res1: Array[Class[?0] forSome { type ?0 }] = Array(                  
  class org.apache.avro.Schema$Parser,                               
  class org.apache.avro.Schema$Field,                                
  class org.apache.avro.Schema$Type,                                 
  class org.apache.avro.JsonProperties$Null                          
```

**Comment by ittayd** [comment]
```
@ import org.apache.avro.Schema.Parser
import org.apache.avro.Schema.Parser
@ new Parser()
cmd2.sc:1: not found: type Parser
val res2 = new Parser()
               ^
Compilation Failed
```

**Comment by ittayd** [comment]
```
> scala -cp C:\Users\idror\.ivy2\cache\org.apache.avro\avro\bundles\avro-1.8.1.jar
Welcome to Scala 2.11.8 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_102).
Type in expressions for evaluation. Or try :help.

scala> new org.apache.avro.Schema.Parser
warning: Class org.codehaus.jackson.JsonFactory not found - continuing with a stub.
warning: Class org.codehaus.jackson.map.ObjectMapper not found - continuing with a stub.
warning: Class org.codehaus.jackson.JsonNode not found - continuing with a stub.
warning: Class org.codehaus.jackson.JsonGenerator not found - continuing with a stub.
warning: Class org.codehaus.jackson.JsonGenerator not found - continuing with a stub.
warning: Class org.codehaus.jackson.JsonNode not found - continuing with a stub.
warning: Class org.codehaus.jackson.JsonNode not found - continuing with a stub.
warning: Class org.codehaus.jackson.JsonNode not found - continuing with a stub.
warning: Class org.codehaus.jackson.JsonNode not found - continuing with a stub.
warning: Class org.codehaus.jackson.JsonNode not found - continuing with a stub.
warning: Class org.codehaus.jackson.JsonNode not found - continuing with a stub.
java.lang.NoClassDefFoundError: org/codehaus/jackson/JsonParseException
  ... 32 elided
Caused by: java.lang.ClassNotFoundException: org.codehaus.jackson.JsonParseException
  at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
  at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
  at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
  ... 32 more

```

**Comment by lihaoyi** [comment]
Looks like a duplicate of #515, and is problem with the latest unstable 2.12 version. This bug doesn't occur in 2.11.x, so one option is to stick with 2.11.x until I manage to sort out the 2.12 problem

**Comment by ittayd** [comment]
I couldn't find instructions on how to use scala 2.11

**Comment by lihaoyi** [comment]
the main instructions install 2.11 http://www.lihaoyi.com/Ammonite/#Ammonite-REPL

**Comment by lihaoyi** [comment]
Fixed in 0.8.1 http://www.lihaoyi.com/Ammonite/#0.8.1
